data/json/items/armor/torso_armor.json

```
@@ -974,22 +974,46 @@
     "repairs_like": "survivor_suit",
     "type": "ARMOR",
     "category": "armor",
-    "name": { "str": "Kevlar vest" },
-    "description": "A lightweight, bullet resistant vest.  Suitable for wearing under clothing.",
-    "weight": "2860 g",
-    "volume": "6 L",
-    "price": 80000,
+    "name": { "str": "Kevlar torso armor" },
```

```suggestion
    "name": { "str": "Kevlar vest" },
```
I think vest works better for this situation. "Torso armor" sounds a bit unnatural.